src/notifications/generate.ts

```diff
@@ -149,12 +150,17 @@ export const generateNotificationMap: Record<
       .avatarManyUsers([ctx.commenter]),
   post_mention: (
     builder,
-    ctx: NotificationPostContext & NotificationDoneByContext,
+    ctx: NotificationPostContext<FreeformPost> & NotificationDoneByContext,
   ) =>
     builder
       .referencePost(ctx.post)
       .icon(NotificationIcon.Comment)
-      .description((ctx.post as WelcomePost).content, true)
+      .description(
+        checkHasMention(ctx.post.title, ctx.doneTo.username)
```

Comment:
Let's add test to cover that, usually we don't check the description but here we have some logic to test

Reply:
I agree. I was looking for some examples and have found it now 🚢